Cr 463

andrewjohnysons opened the pull request:
Added Docker and Travis files
Merged with findQuestionnaireIdByCaseId endpoint
refactored

andrewjohnysons committed:
Changed project name in various places.
Added travis and docker file configs.

andrewjohnysons committed:
Merge branches 'CR-463' and 'master' of github.com:ONSdigital/census-mock-case-api-service into CR-463

 Conflicts:
	src/main/java/uk/gov/ons/ctp/integration/mockcaseapiservice/client/model/CaseContainerDTO.java
	src/main/java/uk/gov/ons/ctp/integration/mockcaseapiservice/client/model/EventDTO.java

andrewjohnysons committed:
Changed project name in various places.
Added travis and docker file configs.

andrewjohnysons committed:
Changed project name in various places.
Added travis and docker file configs.
fixed package errors

cookel2 commented:
The code looks good to me and I have tested that the new endpoint (/ccs/{caseId}/qid) still works correctly in my local environment.

cookel2 reviewed (approved):
The test works correctly and so I approve the changes.

philwhiles committed:
Merge branch 'master' into CR-463

philwhiles reviewed (approved):
OK

philwhiles committed:
CR-463 fmt

andrewjohnysons committed:
Changed project name in various places.
Added travis and docker file configs.
fixed package errors
added kubernetes support

andrewjohnysons committed:
Merge remote-tracking branch 'origin/CR-463' into CR-463
